src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -125,12 +115,22 @@ <h4 class="panel-title border-bottom-small mt" translate>
           <div class="panel-body pt0">
             <ul class="list-unstyled">
               <li>
-                <a class="commentLink" href="" ng-click="showRecipientComments = !showRecipientComments" translate>Send a Message to {{$ctrl.productData.displayName}}</a>
-                <textarea class="staff-comments give-modal-comments" rows="3" ng-model="$ctrl.itemConfig['recipient-comments']" ng-if="showRecipientComments || $ctrl.itemConfig['recipient-comments']" style="display:block;"></textarea>
+                <a class="commentLink" href="" ng-click="showRecipientComments = !showRecipientComments"><translate>Send a Message to</translate> {{$ctrl.productData.displayName}}</a>
```

Comment:
Will the translate directive not work? It seems to [support interpolation](https://angular-gettext.rocketeer.be/dev-guide/annotate/#interpolation).